app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/importlogins/ImportLoginsScreen.kt

```diff
@@ -0,0 +1,240 @@
+        Text(
+            text = stringResource(R.string.give_your_vault_a_head_start),
+            style = BitwardenTheme.typography.titleMedium,
+            color = BitwardenTheme.colorScheme.text.primary,
+            textAlign = TextAlign.Center,
+            modifier = Modifier.standardHorizontalMargin(),
+        )
+        Spacer(Modifier.height(12.dp))
+        Text(
+            text = stringResource(
+                R.string.from_your_computer_follow_these_instructions_to_export_saved_passwords,
+            ),
+            style = BitwardenTheme.typography.bodyMedium,
+            color = BitwardenTheme.colorScheme.text.primary,
+            textAlign = TextAlign.Center,
+            modifier = Modifier.standardHorizontalMargin(),
+        )
+        Spacer(Modifier.height(24.dp))
+        BitwardenFilledButton(
+            label = stringResource(R.string.get_started),
+            onClick = onGetStartedClick,
+            modifier = Modifier
+                .standardHorizontalMargin()
+                .fillMaxWidth(),
+        )
+        Spacer(Modifier.height(12.dp))
+        BitwardenOutlinedButton(
+            label = stringResource(R.string.import_logins_later),
+            onClick = onImportLaterClick,
+            modifier = Modifier
+                .standardHorizontalMargin()
+                .fillMaxWidth(),
+        )
+    }
+}
+
+@Preview
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_YES)
+@Composable
+private fun ImportLoginsInitialContent_preview() {
+    BitwardenTheme {
+        Surface {
+            ImportLoginsContent(
+                onGetStartedClick = {},
+                onImportLaterClick = {},
+            )
+        }
+    }
+}
+
+@Preview
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_YES)
+@Composable
+private fun ImportLoginsScreenDialog_preview(
+    @PreviewParameter(ImportLoginsDialogContentPreviewProvider::class) state: ImportLoginsState,
+) {
+    BitwardenTheme {
+        Surface {
+            ImportLoginsDialogContent(
+                state = state,
+                handler = ImportLoginHandler(
+                    onDismissDialog = {},
+                    onConfirmGetStarted = {},
+                    onConfirmImportLater = {},
+                    onCloseClick = {},
+                    onGetStartedClick = {},
+                    onImportLaterClick = {},
+                ),
+            )
+            ImportLoginsContent(
+                onGetStartedClick = {},
+                onImportLaterClick = {},
+            )
+        }
+    }
+}
+
+private class ImportLoginsDialogContentPreviewProvider :
+    PreviewParameterProvider<ImportLoginsState> {
+    override val values: Sequence<ImportLoginsState>
+        get() = sequenceOf(
+            ImportLoginsState(
+                dialogState = ImportLoginsState.DialogState.GetStarted,
+            ),
+            ImportLoginsState(
+                dialogState = ImportLoginsState.DialogState.ImportLater,
```

Comment:
Does this buy us much?

Reply:
saves repeating the above function for the two states, does it create any problem other than we don't have it other places?